app/models/audit/AuditTaskInteractionTable.scala

```diff
@@ -261,6 +261,218 @@ object AuditTaskInteractionTable {
+    ).first
+  }
+
+  /**
+   * Calculate combined time spent validating for all users in the past 24 hours.
+   */
+  def countTodayValidating(): Float = db.withSession { implicit session =>
+    Q.queryNA[Float](
+      s"""SELECT CAST(extract(second from SUM(diff)) / 60 +
+          |            extract(minute from SUM(diff)) +
+          |            extract(hour from SUM(diff)) * 60 AS decimal(10,2)) / 60.0 AS hours_validated
+          |FROM (
+          |    SELECT (timestamp - LAG(timestamp, 1) OVER(ORDER BY timestamp)) AS diff
+          |    FROM validation_task_interaction
+          |    WHERE (timestamp AT TIME ZONE 'US/Pacific') > (NOW() AT TIME ZONE 'US/Pacific')
+          |) "time_diffs"
+          |WHERE diff < '00:05:00.000' AND diff > '00:00:00.000';""".stripMargin
+    ).first
+  }
+
+  /**
+    * Calculate combined time spent validating for all users in the past week.
+    */
+  def countPastWeekValidating(): Float = db.withSession { implicit session =>
+    Q.queryNA[Float](
+      s"""SELECT CAST(
+         |  extract(second from SUM(diff)) / 60 +
+         |  extract(minute from SUM(diff)) +
+         |  extract(hour from SUM(diff)) * 60 AS decimal(10,2)) / 60.0 AS hours_validated
+         |FROM (
+         |  SELECT (timestamp - LAG(timestamp, 1) OVER(ORDER BY timestamp)) AS diff
+         |  FROM validation_task_interaction
+         |  WHERE (timestamp AT TIME ZONE 'US/Pacific') > (NOW() AT TIME ZONE 'US/Pacific') - interval '168 hours'
+         |) "time_diffs"
+         |WHERE diff < '00:05:00.000' AND diff > '00:00:00.000';""".stripMargin
+    ).first
+  }
+
+  /**
+    * Calculates the median auditing time per 100 meters for all users who have audited
+    * at least 100 meters and spent 30 minutes auditing.
+    */
+  def countTotalAverageAuditingTime(): Float = db.withSession { implicit session =>
+    Q.queryNA[Float](
+      s"""SELECT percentile_CONT(0.5) WITHIN GROUP (ORDER BY minutes_per_100m)
+         |FROM (
+         |    SELECT user_stat.user_id,
+         |           minutes_audited / (meters_audited / 100) AS minutes_per_100m
+         |    FROM user_stat
+         |    INNER JOIN (
+         |        SELECT user_id,
+         |               CAST(
+         |                 extract(second from SUM(diff)) / 60 +
+         |                 extract(minute from SUM(diff)) +
+         |                 extract(hour from SUM(diff)) * 60 AS decimal(10,2)) AS minutes_audited
+         |        FROM (
+         |            SELECT audit_task.user_id, timestamp,
+         |                   (timestamp - LAG(timestamp, 1) OVER(PARTITION BY audit_task.user_id ORDER BY timestamp)) AS diff
+         |            FROM audit_task_interaction_small
+         |            INNER JOIN audit_task ON audit_task_interaction_small.audit_task_id = audit_task.audit_task_id
+         |        ) "time_diffs"
+         |        WHERE diff < '00:00:30.000' AND diff > '00:00:00.000'
```

Comment:
This should still be set to 5 minutes. This part of the query is looking for consecutive interactions that are less than 5 minutes apart. If it's less than 5 min apart, we assume that they are auditing continuously. If you switch it to 30 minutes, they could take a 25 minute break and we would consider them as working that whole time!